python/api/cluster/views.py

```diff
@@ -251,7 +252,7 @@ def post(self, request, cluster_id, upgrade_id):
         return create(serializer, upgrade_id=int(upgrade_id), obj=cluster)
 
 
-class StatusList(GenericAPIPermStatusView, InterfaceView):
+class StatusList(GenericAPIView, InterfaceView):
```

Comment:
Using GenericAPIView instead of own GenericAPIPermView means access for unauthorized - am I getting that right?

Reply:
Using GenericApiView is mean using api_settings.DEFAULT_PERMISSION_CLASSES. In our case it is class IsAuthenticated. Manual testing in API shows that unathorized user getting 401 Unathorized error.